lib/eslint.js

```diff
@@ -33,6 +33,18 @@ const assert = require("assert"),
 //------------------------------------------------------------------------------
 
 /**
+ * The result of a parsing operation from parseForESLint()
+ * @typedef {Object} CustomParseResult
+ * @property {Object} ast The ESTree AST Program node.
+ * @property {Object} services An object containing additional services related
+ *      to the parser.
+ */
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
```

Comment:
The "Helpers" block is listed above the JSDoc typedef as well-- should that first block say something like "JSDoc Typedefs" instead?